hmcts/pre-api
# Call stopLiveEvent method for all running live events

### JIRA ticket(s)

- S28-2974

### Change description

- Update to the cron to call the method to stop all live events currently found in AMS/MK

> [!IMPORTANT]
> This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

**QA instructions**

- This is not something which can be easily called remotely to test it. The simplest way is to check out the code and run it yourself or observe the results after I've run it.
- Create 1 or more live events in Staging AMS
- Call `TASK_NAME=CleanupLiveEvents ./gradlew bootRun`
- Observe changes in AMS and updated records in the db

Note: If this PR is merged, it will start running at 8pm every day. This should mean the sweeper flow can be removed but should be planned properly prior to merging this PR.
